sdks/python/apache_beam/test_pipeline.py

```diff
@@ -54,23 +55,54 @@ class TestPipeline(Pipeline):
 
   def __init__(self, runner=None, options=None, argv=None):
     if options is None:
-      options = self.create_pipeline_opt_from_args()
+      options = self.create_test_pipeline_options()
```

Comment:
could this be just:

`options = PipelineOptions(self.get_test_option_args(argv))` and remove `create_test_pipeline_options()`